functions-app/census-import/census-import.data.service.js

```diff
@@ -0,0 +1,100 @@
+const mssql = require('mssql') /* special */
+const config = require('../config')
+const R = require('ramda')
+
+let pool
+
+module.exports.initPool = async function initPool (context) {
+  const poolConfig = {
+    database: config.Sql.Database,
+    server: config.Sql.Server,
+    port: config.Sql.Port,
+    requestTimeout: 10 * 60 * 1000,
```

Comment:
please can you move this to a config variable with a default value the same as whats in admin app.

i am currently tweaking these in load tests and it has a profound effect on the database load behaviour when coupled with the retry mechanism

Reply:
I am happy to move it to a config variable, but the current 10-minute request timeout may be too short, rather than too long.   The default request timeout for the functions-app is five minutes which is too short.

I don't see how this function is affected by the load tests - pupil load takes place before any other load is placed on the system.

Reply:
the default value of 10 minutes is ok, its just about having it configurable in case we need to support production.  we don't want to have to redeploy.  and yes, this is literally a 'run once before we go live' thing

Reply:
Ok.  It will almost certainly take longer than 10 mins to run the main SQL, so it's set accordingly.